tools/processresults.py

```diff
@@ -23,7 +24,9 @@
 ATTRIBUTE_QUESTION_PREFIX = 'To what degree do each of the following'
 
 
-def ProcessResults(json_in_file, csv_prefix):
+def ProcessResults(json_in_file, csv_prefix,
+                   start_date = FRIENDS_AND_FAMILY_BETA_DATE,
```

Comment:
nit: Could |start_date| be renamed to |filter_items_before_date| or something like that? Same for and |demographic_start_date|.